core/templates/components/rubrics-editor/rubrics-editor.component.html

```diff
@@ -65,20 +65,32 @@
             </button>
             <button type="button"
                     class="btn btn-success protractor-test-save-rubric-explanation-button"
-                    [disabled]="!this.isExplanationValid(this.rubric.getDifficulty(), idx)"
+                    [disabled]="!this.isExplanationEmpty(explanation)
+                      || !this.isExplanationLengthValid(this.rubric.getDifficulty(), idx)"
                     (click)="this.saveExplanation(this.rubric.getDifficulty(), idx)">
               Save
             </button>
           </div>
+          <br>
+          <div *ngIf="!this.isExplanationLengthValid(this.rubric.getDifficulty(), idx)" class="rubric-error-message">
+            Explanations should have at most 300 characters.
+          </div>
+          <div class="oppia-card-content-limit-error" *ngIf="this.isTotalExplanationsLengthReached()">
+            There can be maximum of 10 explanations across all levels.
+          </div>
```

Comment:
Move this div outside the `*ngFor` statement, since we only want to show it once, and not 10 times.

Reply:
Don't you think it's not going to repeat 10 times as we have the condition in `*ngIf`(checks the total number of explanations), the reason I put it there is because on the outer side it checks if the editor is open and this warning will be shown when the editor is open and if we take that out of that condition then it will be shown all the time. Please do let me know if we need to change this.
Thanks

Reply:
Ah okay you're right! No need to change anything.